packages/core/include/combo/oot/actors/Custom_Torch2.h

```diff
@@ -0,0 +1,32 @@
+#ifndef Z_EN_TORCH2_H
+#define Z_EN_TORCH2_H
+
+#include <combo/common/actor.h>
+
+struct EnTorch2;
+
+typedef struct EnTorch2 {
```

Comment:
Please respect the style conventions: curly braces on their line

Reply:
We don't have nor want a formater. You're free to use whatever you want locally.